README.md

```diff
@@ -14,18 +14,29 @@ Requirements:
 
 ### Installation
 
+
+#### NPM package
+
 We recommend to use `yarn` as your package manager, and also a build tool like webpack or browserify in order to use it.
 
 ```bash
 yarn add @typeform/embed
 ```
+The NPM version doesn't include neither React, ReacDOM nor polyfills. It's a responsibility of the application to provide those deps as peer dependencies and control the environment polyfilling process.
+
+#### CDN script
 
-If you're not, you can still use it with our [CDN](https://embed.typeform.com/embed.js):
+You can also consume the library from our [CDN](https://embed.typeform.com/embed.js):
 
 ```html
 <script src="https://embed.typeform.com/embed.js"></script>
 <!-- this script exposes the API to `window.typeformEmbed` -->
 ```
+The CDN version has everything bundled, so the script is ready to use out-of the box. It contains react, react-dom, and polyfills. If you believe everything is already injected in your environment globals you can use a more lightweight version, ***although it's not recommended***:
```

Comment:
Just to reiterate a bit on what we already commented on other occasions. I believe that we should be able to get rid of React and React DOM. It seems that the functionality provided by the embed SDK could be implemented without these third-party libraries. Not sure if we already have a ticket for this, or if you think we should create one. WDYT @prilutskiy @mathio ?

Reply:
I agree however that would require rewrite of the library. We do not just use JSX but also life cycle methods. It is out of scope for now but definitely something we should revisit later.

Reply:
Absolutely agree with you guys, that's a great point! Let's brainstorm this and analyze the cost/benefits.